app/api/public/__init__.py

```diff
@@ -0,0 +1,7 @@
+from fastapi import APIRouter
+
+from . import cheesegull
+
+public_router = APIRouter()
```

Comment:
After this is merged, I will change nginx to expose this at the beatmaps.akatsuki.gg/ root

Reply:
And then I will remove the old routes

Reply:
https://github.com/osuAkatsuki/nginx-conf/commit/020a0b0bf95cd508480d87e6aa585f3542495801